### PR TITLE
Bubblez.app Website

### PR DESCRIPTION
**Pull request by ProjectVexion**

### CODE REVIEW

**Comment by ProjectVexion** [comment]
image
image

File: websites/B/Bubblez.app/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "url": [
+    "bubblez.app",
+	"canary.bubblez.app"
+  ],
+  "description": {
+    "en": "The best upcoming Social Media site."
+  },
+  "service": "Bubblez.app",
+  "version": "1.1.6",
```
Comment (ririxidev) [inline websites/B/Bubblez.app/dist/metadata.json:25]: 1.0.0

File: websites/B/Bubblez.app/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "contributors": [
+    {
+      "name": "embed",
+      "id": "476641014841475084"
+    },
+    {
+      "name": "bob5918",
+      "id": "316673724990488577"
+    }
+  ],
```
Comment (ririxidev) [inline websites/B/Bubblez.app/dist/metadata.json:16]: they didn't contribute to this presence in commits (also it is release so delete them)

File: websites/B/Bubblez.app/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "bubblez.app",
+	"canary.bubblez.app"
+  ],
+  "description": {
+    "en": "The best upcoming Social Media site."
+  },
+  "service": "Bubblez.app",
+  "version": "1.1.6",
+  "logo": "https://cdn.bubblez.app/official/reBrand-test.png",
+  "thumbnail": "https://cdn.bubblez.app/official/reBrand-test.png",
```
Comment (ririxidev) [inline websites/B/Bubblez.app/dist/metadata.json:27]: we accept only imgur links :) *i.imgur.com*

File: websites/B/Bubblez.app/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  "color": "#00CC99",
+  "tags": [
+    "Social",
+    "Media",
+    "Social Media"
```
Comment (ririxidev) [inline websites/B/Bubblez.app/dist/metadata.json:32]: no spaces in tags

File: websites/B/Bubblez.app/presence.ts [hunk 5]
```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+    clientId: "751981945881231430" //The client ID of the Application created at https://discordapp.com/developers/applications
```
Comment (ririxidev) [inline websites/B/Bubblez.app/presence.ts:2]: delete this comment

File: websites/B/Bubblez.app/presence.ts [hunk 6]
```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+    clientId: "751981945881231430" //The client ID of the Application created at https://discordapp.com/developers/applications
+});
+
+const capitalize = (text: string): string => {
```
Comment (ririxidev) [inline websites/B/Bubblez.app/presence.ts:5]: merge consts

File: websites/B/Bubblez.app/presence.ts [hunk 7]
```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+    clientId: "751981945881231430" //The client ID of the Application created at https://discordapp.com/developers/applications
+});
+
+const capitalize = (text: string): string => {
+  var texts = text.replace(/[[{(_)}\]]/g, " ").split(" ");
```
Comment (ririxidev) [inline websites/B/Bubblez.app/presence.ts:6]: no vars, change to const

File: websites/B/Bubblez.app/presence.ts [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+	}
+	
+	
+	const PresenceData: PresenceData = {
+        largeImageKey: largeImageKey, /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageKey: smallImageKey, /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageText: smallImageText, //The text which is displayed when hovering over the small image
+        details: details, //The upper section of the presence text
+		state: state //"Reading some posts" //The lower section of the presence text
+
```
Comment (ririxidev) [inline websites/B/Bubblez.app/presence.ts:100]: please change it all... For now it is so low quality and for now we are not going to accept this presence in this state.
Also no vars.

**Review by ririxidev — CHANGES_REQUESTED** [review]
fix them and open new pr because it is low quality presence for now.

**Pull request closed without merging**